plugin-testing-core/src/test/java/com/palantir/gradle/plugintesting/TestDependencyVersionsTests.java

```diff
@@ -19,15 +19,30 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 public class TestDependencyVersionsTests {
-    private static final String SAMPLE_VERSIONS = "foo:bar:100,com.palantir:gradle-plugin-testing:1.2.3";
+    private static final String SAMPLE_VERSIONS = "foo:bar=100\ncom.palantir:gradle-plugin-testing=1.2.3";
```

Comment:
nit: let's use text blocks!
```suggestion
    private static final String SAMPLE_VERSIONS = """
        foo:bar=100
        com.palantir:gradle-plugin-testing=1.2.3
      """;
```